packages/material-tree-renderer/src/services/property.util.ts

```diff
@@ -1,17 +1,14 @@
 import * as _ from 'lodash';
-import { JsonSchema7 } from '@jsonforms/core';
+import { JsonSchema } from '@jsonforms/core';
```

Comment:
Are we using version 4? If yes, this might cause inconsistencies for the applications that depend on material-tree-renderer. As I remembered using version 7 in ui-schema-editor with material-tree-renderer which depends on version 4 was problematic.